app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -3305,3 +3308,22 @@ mean other gateways, load balancers, NATs, and such in front of Kong. If there i
 
 - [port maps](/gateway/latest/reference/configuration/#port_maps)
 - `X-Forwarded-*` headers
+
+---
+
+## Changelog
+
+### 2.2.0
+
+* Starting with {{site.base_gateway}} 2.7.0.0, if keyring encryption is enabled,
+ the `client_id`, `client_secret`, `session_auth`, and `session_redis_auth`
+ parameter values will be encrypted.
+
+  Additionally, the `d`, `p`, `q`, `dp`, `dq`, `qi`, `oth`, `r`, `t`, and `k`
+  fields inside `openid_connect_jwks.previous[...].` and `openid_connect_jwks.keys[...]`
+  will be marked as encrypted.
+
+  {:.important}
+  > There's a bug in {{site.base_gateway}} that prevents keyring encryption
+  from working on deeply nested fields, so the `encrypted=true` setting does not
+  currently have any effect on the nested fields in this plugin.
```

Comment:
@kikito Is this accurate for the OIDC plugin? Looks like it should have the same issues as JWT signer.

Reply:
Correct, it will have the same issues. Well spotted.